README.md

```diff
@@ -1 +1,34 @@
 # SOM_Weather_Forecast_API
+
+###Description 
+This weather forecast application was created to conduct unit tests on the weather Api.
```

Comment:
These are not unit tests and it is not a application. We have created a SOM to enable us to easily write integration / api tests.